PR addressing issue958

riccardoAlbertoni opened the pull request:
Adding missing language tags to WD examples #958

riccardoAlbertoni committed:
adding language tags in the examples 1-4

riccardoAlbertoni committed:
adding language tags to examples 5-12

riccardoAlbertoni committed:
adding language tags to examples 13-18

riccardoAlbertoni committed:
adding language tags to examples till 45 in WD

riccardoAlbertoni committed:
adding language tags to examples till 52 in WD

riccardoAlbertoni committed:
adding two missing language tags to examples 49 in WD

riccardoAlbertoni committed:
minor adjustement

riccardoAlbertoni committed:
adding language tags to ttl files in the directory examples

riccardoAlbertoni committed:
adding language tags to ttl files in the directory examples

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -232,13 +232,13 @@ dap:atnf-P366-2003SEPT_1
       dct:creator <https://www.doi.org/> ;
       skos:notation "10.4225/08/598dc08d07bb7" ;
-      adms:schemeAgency "International DOI Foundation" ;
+      adms:schemeAgency "International DOI Foundation"en ;
```
```suggestion
      adms:schemeAgency "International DOI Foundation"@en ;
```

riccardoAlbertoni committed:
Update dcat/examples/csiro-dap-examples.ttl

riccardoAlbertoni committed:
Merge branch 'gh-pages' into dcat-issue958-riccardo

riccardoAlbertoni committed:
Merge branch 'dcat-issue958-riccardo' of https://github.com/w3c/dxwg into dcat-issue958-riccardo

riccardoAlbertoni committed:
adding language tags to ttl files in the directory examples

riccardoAlbertoni commented:
I have added the language tags in all the examples in the DCAT WD except when  IMO adding the language tag does not make much sense, such as  
-  rdf:value in example 31.
-  rdf:value and XMLLiteral in examples 39 and 40 -
- dct:type "Observation"  and dct:bibliographicCitation in example 46 

I have also added language tags to TTL file contained in the directory of examples. 
Please @dr-shorthair check if this makes sense in your examples.  I guess the json-ld examples were not manually curated and so we can obtain the update simply converting from the TTL.  Is this ok?

dr-shorthair commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -232,13 +232,13 @@ dap:atnf-P366-2003SEPT_1
       dct:creator <https://www.doi.org/> ;
       skos:notation "10.4225/08/598dc08d07bb7" ;
-      adms:schemeAgency "International DOI Foundation" ;
+      adms:schemeAgency "International DOI Foundation"@en ;
```
This is a proper name so I don't think it should have a language-tag

riccardoAlbertoni replied:
```suggestion
      adms:schemeAgency "International DOI Foundation";
```

dr-shorthair commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -361,21 +361,21 @@ dap:d33937
-  dcat:keyword "reference system" ;
-  dcat:keyword "stratigraphy" ;
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
```
Several of these are acronyms or proper names, so not language-specific

riccardoAlbertoni replied:
```suggestion
  dcat:keyword "GeoSPARQL" ;
```

dr-shorthair commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
-  dcat:keyword "reference system" ;
-  dcat:keyword "stratigraphy" ;
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
```
Proper names and acronyms

riccardoAlbertoni replied:
```suggestion
  dcat:keyword "GeoSPARQL" ;
```

dr-shorthair commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -568,5 +568,5 @@ dap:doi-P366-2003SEPT
   dct:creator <https://researchdata.ands.org.au/> ;
   skos:notation "10.4225/08/598dc08d07bb7" ;
-  adms:schemeAgency "International DOI Foundation" ;
+  adms:schemeAgency "International DOI Foundation"@en ;
```
proper name

riccardoAlbertoni replied:
```suggestion
  adms:schemeAgency "International DOI Foundation" ;
```

dr-shorthair commented on dcat/examples/ga-courts.ttl:
```diff
@@ -18,28 +18,28 @@
-  rdfs:label "Commonwealth of Australia (Geoscience Australia)" ;
-  v:country-name "Australia" ;
+  rdfs:label "Commonwealth of Australia (Geoscience Australia)"@en;
+  v:country-name "Australia"@en;
```
I don't think the values in a vCard are language-specific strings

riccardoAlbertoni replied:
```suggestion
  v:country-name "Australia" ;
```

dr-shorthair commented on dcat/examples/ga-courts.ttl:
```diff
@@ -128,26 +128,26 @@ ga-courts:jc-wfs
-  dcat:keyword "web service" ;
+  dcat:keyword "Australia"@en;
+  dcat:keyword "Published_External"@en;
+  dcat:keyword "WFS"@en;
```
Acronym

riccardoAlbertoni replied:
```suggestion
  dcat:keyword "WFS" ;
```

dr-shorthair commented on dcat/examples/ga-courts.ttl:
```diff
@@ -95,27 +95,27 @@ ga-courts:jc-esri
-  dcat:keyword "web mapping" ;
-  dcat:keyword "web service" ;
+  dcat:keyword "Australia"@en;
+  dcat:keyword "MapServer"@en;
```
Proper name

riccardoAlbertoni replied:
```suggestion
  dcat:keyword "MapServer" ;
```

dr-shorthair commented on dcat/examples/ga-courts.ttl:
```diff
@@ -77,15 +77,15 @@ ga-courts:jc
-  dcat:keyword "risk assessment" ;
+  dcat:keyword "AU"@en;
+  dcat:keyword "Database"@en;
+  dcat:keyword "GIS"@en;
```
Acronym

riccardoAlbertoni replied:
```suggestion
  dcat:keyword "GIS" ;
```

dr-shorthair commented on dcat/examples/ga-courts.ttl:
```diff
@@ -167,18 +167,18 @@ ga-courts:jc-wms
-  dcat:keyword "web service" ;
+  dcat:keyword "Australia"@en;
+  dcat:keyword "Published_External"@en;
+  dcat:keyword "WMS"@en;
```
Acronym

riccardoAlbertoni replied:
```suggestion
  dcat:keyword "WMS" ;
```

dr-shorthair commented on dcat/examples/ga-wms.ttl:
```diff
@@ -16,24 +16,24 @@
-  dcat:keyword "DEM" ;
-  dcat:keyword "ECOLOGY Landscape" ;
-  dcat:keyword "LAND Topography Models" ;
+  dcat:keyword "DEM"@en ;
```
Acronym

riccardoAlbertoni replied:
```suggestion
  dcat:keyword "DEM" ;
```

makxdekkers commented:
@riccardoAlbertoni I think "Observation" as value for dct:type in example 46 is incorrect. Object of dct:type should not be a literal.

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -361,21 +361,21 @@ dap:d33937
-  dcat:keyword "stratigraphy" ;
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
```
```suggestion
  dcat:keyword "OWL" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -361,21 +361,21 @@ dap:d33937
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
+  dcat:keyword "OWL-Time"@en ;
```
```suggestion
  dcat:keyword "OWL-Time" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -361,21 +361,21 @@ dap:d33937
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
+  dcat:keyword "OWL-Time"@en ;
+  dcat:keyword "RDF"@en ;
```
```suggestion
  dcat:keyword "RDF" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -361,21 +361,21 @@ dap:d33937
+  dcat:keyword "OWL"@en ;
+  dcat:keyword "OWL-Time"@en ;
+  dcat:keyword "RDF"@en ;
+  dcat:keyword "SOSA"@en ;
```
```suggestion
  dcat:keyword "SOSA" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -361,21 +361,21 @@ dap:d33937
+  dcat:keyword "OWL-Time"@en ;
+  dcat:keyword "RDF"@en ;
+  dcat:keyword "SOSA"@en ;
+  dcat:keyword "SSN"@en ;
```
```suggestion
  dcat:keyword "SSN" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
-  dcat:keyword "stratigraphy" ;
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
```
```suggestion
  dcat:keyword "OWL" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
-  dcat:keyword "vocabulary" ;
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
+  dcat:keyword "OWL-Time"@en ;
```
```suggestion
  dcat:keyword "OWL-Time" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
+  dcat:keyword "GeoSPARQL"@en ;
+  dcat:keyword "OWL"@en ;
+  dcat:keyword "OWL-Time"@en ;
+  dcat:keyword "RDF"@en ;
```
```suggestion
  dcat:keyword "RDF" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
+  dcat:keyword "OWL"@en ;
+  dcat:keyword "OWL-Time"@en ;
+  dcat:keyword "RDF"@en ;
+  dcat:keyword "SOSA"@en ;
```
```suggestion
  dcat:keyword "SOSA" ;
```

riccardoAlbertoni commented on dcat/examples/csiro-dap-examples.ttl:
```diff
@@ -548,16 +548,16 @@ dap:d33937_1
+  dcat:keyword "OWL-Time"@en ;
+  dcat:keyword "RDF"@en ;
+  dcat:keyword "SOSA"@en ;
+  dcat:keyword "SSN"@en ;
```
```suggestion
  dcat:keyword "SSN" ;
```

riccardoAlbertoni commented on dcat/examples/ga-courts.ttl:
```diff
@@ -77,15 +77,15 @@ ga-courts:jc
-  dcat:keyword "infrastructure" ;
-  dcat:keyword "risk analysis" ;
-  dcat:keyword "risk assessment" ;
+  dcat:keyword "AU"@en;
```
```suggestion
  dcat:keyword "AU" ;
```

riccardoAlbertoni commented on dcat/index.html:
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
   skos:notation "https://doi.org/10.5281/zenodo.1486279"^^xsd:anyURI;
   # the authority/agency defining the identifier scheme, used if the agency has no URI
-  adms:schemaAgency "International DOI Foundation" ;
+  adms:schemaAgency "International DOI Foundation"@en ;
```
```suggestion
  adms:schemaAgency "International DOI Foundation" ;
```

riccardoAlbertoni commented on dcat/index.html:
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
 
 ex:InternationalDOIFundation a foaf:Organization ;
-  rdfs:label "International DOI Foundation";
+  rdfs:label "International DOI Foundation"@en;
```
```suggestion
  rdfs:label "International DOI Foundation" ;
```

riccardoAlbertoni commented on dcat/index.html:
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
 
 &lt;https://example.org/PoelenJorritH&gt; a foaf:Person;
-  foaf:name "Jorrit H. Poelen" ;
+  foaf:name "Jorrit H. Poelen"@en ;
```
```suggestion
  foaf:name "Jorrit H. Poelen" ;
```

riccardoAlbertoni commented on dcat/index.html:
```diff
@@ -2993,25 +2993,25 @@ <h2>Dereferenceable identifiers</h2>
   skos:notation "https://orcid.org/0000-0003-3138-4118"^^xsd:anyURI ;
   # the authority/agency defining the identifier scheme, used if the agency has no URI
-  adms:schemaAgency "ORCID" ;
+  adms:schemaAgency "ORCID"@en ;
```
```suggestion
  adms:schemaAgency "ORCID" ;
```

riccardoAlbertoni commented on dcat/index.html:
```diff
@@ -3047,11 +3047,11 @@ <h2>Indicating common identifier types</h2>
   skos:notation "PA 1-060-815"^^ex:type ;
   # Human readable schema agency
-  adms:schemaAgency "US Copyright Office" ;
+  adms:schemaAgency "US Copyright Office"@en ;
```
```suggestion
  adms:schemaAgency "US Copyright Office" ;
```

riccardoAlbertoni committed:
Update dcat/examples/csiro-dap-examples.ttl

riccardoAlbertoni committed:
Update dcat/examples/csiro-dap-examples.ttl

riccardoAlbertoni committed:
Update dcat/examples/ga-courts.ttl

riccardoAlbertoni committed:
Apply suggestions from code review

Removing language tag for Acronyms, proper name, agency as suggested by @dr-shorthair

riccardoAlbertoni committed:
Update dcat/index.html

riccardoAlbertoni committed:
Update dcat/index.html

riccardoAlbertoni committed:
Update dcat/index.html

riccardoAlbertoni committed:
Update dcat/index.html

riccardoAlbertoni committed:
Update dcat/index.html

dr-shorthair commented:
@makxdekkers I was responsible for that. 
Yes, a URI is always preferable, though `dct:type` is a `rdf:Property` so a literal is allowed. 

There are a lot of options that might be OK 

1. http://www.opengis.net/def/observationType/OGC-OM/2.0/OM_Observation
2. http://www.w3.org/ns/sosa/Observation 
3. http://purl.obolibrary.org/obo/BCO_0000003 
4. http://dbpedia.org/resource/Observation

Which would you prefer? Note that 2. and 3. are formal OWL Classes, so probably would be the value of rdf:type rather than dct:type, and therefore might not be appropriate here as they imply much narrower semantics. 1. is similarly tied to a specific UML model. So maybe dbpedia is best?

makxdekkers commented:
No. 4, Dbpedia sounds good.

Commit:
replaced "Observation" with <http://dbpedia.org/resource/Observation>
